src/redux/reducers/user.ts

```diff
@@ -1,36 +1,39 @@
 import { UserAction, UserState } from "../../../types";
 
 // Initialize initial User State
-// TODO: 
+// TODO:
 // - Check mobile cache storage for access_token
 // - Attempt to authenticate user
 // - else initialize with initialState
 const initialState: UserState = {
-    user: {
-        id: 1991,
-        username: "",
-        image_url: ""
-    },
-    isLoggedIn: false
-}
+  user: {
+    id: "1991",
+    username: "",
+    image_url: "",
+  },
+  isLoggedIn: false,
```

Comment:
This is a mistake.

Reply:
Do you plan on reverting to the original? Is ID a number or string?